doc/plugins/presale-saml.rst

```diff
@@ -0,0 +1,405 @@
+        attribute. Often starting with ``urn:mace:dir:attribute-def:``,
+        ``urn:oid:`` or ``http://``/``https://``.
+      * ``nameFormat``: String describing the type of ``name`` that has been
+        set in the previous section. Often starting with
+        ``urn:mace:shibboleth:1.0:`` or ``urn:oasis:names:tc:SAML:2.0:``.
+
+    Your IdP can provide you with a list of available attributes. See below
+    for a sample configuration in an academic context.
+
+    Note, that you can have multiple attributes with the same ``friendlyName``
+    but different ``name``s. This is often used in systems, where the same
+    information (for example a persons name) is saved in different fields -
+    for example because one institution is returning SAML 1.0 and other
+    institutions are returning SAML 2.0 style attributes. Typically, this only
+    occurs in mix environments like the DFN-AAI with a large number of
+    participants. If you are only using your own institutions IdP and not
+    authenticating anyone outside of your realm, this should not be a common
+    sight.
+
+``Encrypt/Sign/Require ...``
+    Does what is says on the box - please inquire with your IdP for the
+    necessary settings. Most settings can be turned on as they increase security,
+    however some IdPs might stumble over some of them.
+
+``Signature / Digest Algorithm``
+    Please chose appropriate algorithms, that both pretix/your SP and the IdP
+    can communicate with. A common source of issues when connecting to a
+    Shibboleth-based IdP is the Digest Algorithm: pretix does not support
+    ``http://www.w3.org/2009/xmlenc11#rsa-oaep`` and authentication will fail
+    if the IdP enforces this.
+
+``Technical/Support Contacts``
+    Those contacts are encoded into the SPs public meta data and might be
+    displayed to users having trouble authenticating. It is recommended to
+    provide a dedicated point of contact for technical issues, as those will
+    be the ones to change the configuration for the SP.
+
+Event / Authentication configuration
+------------------------------------
+
+Basic settings
+^^^^^^^^^^^^^^
+
+Once the plugin has been enabled for a pretix event using the Plugins-menu from
+the event's settings, a new *SAML* menu item will show up.
+
+On this page, the actual authentication can be configured.
+
+``Checkout Explanation``
+    Since most users probably won't be familiar with why they have to authenticate
+    to buy a ticket, you can provide them a small blurb here. Markdown is supported.
+
+``Attribute RegEx``
+    By default, any successful authentication with the IdP will allow the user to
+    proceed with their purchase. Should the allowed audience needed to be restricted
+    further, a set of regular Expressions can be used to do this.
+
+    An Attribute RegEx of ``{}`` will allow any authenticated user to pass.
+
+    A RegEx of ``{ "affiliation": "(employee@pretix.eu|staff@pretix.eu)" }`` will
```

Comment:
The example should probably be this?

```suggestion
    A RegEx of ``{ "affiliation": "^(employee@pretix.eu|staff@pretix.eu)$" }`` will
```

Reply:
It actually also works without the ^ and $... Just tried it out again...

Reply:
Did you just try that someone with ``staff@pretix.eu`` gets in correctly or did you also try if someone affiliated with ``staff@pretix.eu.myevildomain.com`` is actually blocked?

Reply:
Oh, I see your point now... I did indeed only test the first thing...

Adding this here *and* updating the plugin itself, too.